vision/common/bounding_box.py

```diff
@@ -0,0 +1,325 @@
+"""
+Bounding box objects represent an area in an image and
+are used to convey information between flight and vision processes.
+"""
+
+from enum import Enum
+
+from typing import Any
+
+import numpy as np
+
+
+class ObjectType(Enum):
+    """
+    Type of object that a BoundingBox represents.
+    """
+
+    STD_OBJECT: str = "std_object"
+    EMG_OBJECT: str = "emg_object"
+    TEXT: str = "text"
+
+
+class BoundingBox:
```

Comment:
Create a function that can take a bounding box defined by (x,y) and (w,h) and convert it to the four corners, and a function that returns the (x,y) and (w,h) representation